dali/operators/numba_function/numba_func.h

```diff
@@ -55,6 +107,10 @@ class NumbaFuncImpl : public Operator<Backend> {
   std::vector<uint64_t> input_shape_ptrs_;
```

Comment:
Shouldn't this be 
```suggestion
  std::vector<uintptr_t> input_shape_ptrs_;
```
?

Reply:
done